modules/apps/message-boards/message-boards-test/src/testIntegration/java/com/liferay/message/boards/internal/upgrade/v6_6_0/test/MBCategoryExternalReferenceCodeUpgradeProcessTest.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.message.boards.internal.upgrade.v6_6_0.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.message.boards.constants.MBCategoryConstants;
+import com.liferay.message.boards.model.MBCategory;
+import com.liferay.message.boards.service.MBCategoryLocalService;
+import com.liferay.message.boards.service.MBCategoryLocalServiceUtil;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.model.ExternalReferenceCodeModel;
+import com.liferay.portal.kernel.test.util.RandomTestUtil;
+import com.liferay.portal.kernel.version.Version;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.upgrade.registry.UpgradeStepRegistrator;
+import com.liferay.portal.upgrade.test.util.BaseExternalReferenceCodeUpgradeProcessTestCase;
+
+import org.junit.runner.RunWith;
+
+/**
+ * @author Marco Galluzzi
+ */
+@RunWith(Arquillian.class)
+public class MBCategoryExternalReferenceCodeUpgradeProcessTest
+	extends BaseExternalReferenceCodeUpgradeProcessTestCase {
+
+	@Override
+	protected ExternalReferenceCodeModel[] addExternalReferenceCodeModels(
+			String tableName)
+		throws PortalException {
+
+		return new ExternalReferenceCodeModel[] {
+			MBCategoryLocalServiceUtil.addCategory(
```

Comment:
I prefer to use injected service instead of Util class as we are doing in some lines below _mbCategoryLocalService.fetchMBCategory(

Reply:
Good catch. I've changed it in [661a29c](https://github.com/liferay-content-management/liferay-portal/pull/5866/commits/661a29c31a9c7ef231a81a2962118d5e70145782)